app/vmagent: fix creating target id if `--promscrape.dropOriginalLabels` flag was used (#4616)
VictoriaMetrics/VictoriaMetrics: app/vmagent/README.md

  This relabeling is used for modifying labels in discovered targets and for dropping unneeded targets.
  See [relabeling cookbook](https://docs.victoriametrics.com/relabeling.html) for details.

>   This relabeling can be debugged by clicking the `debug` link (this link will be unavailable if `--promscrape.dropOriginalLabels` flag enabled) at the corresponding target on the `http://vmagent:8429/targets` page

Reviewer: ```suggestion
  This relabeling can be debugged by clicking the `debug` link (the link is available only if `--promscrape.dropOriginalLabels=false`) at the corresponding target on the `http://vmagent:8429/targets` page
```
